source/projects/create-project-loading-schema-files.txt

```diff
@@ -103,6 +103,11 @@ to generate DDL files from your relational system.
 
       #. Select :guilabel:`Options` and enable :guilabel:`Include CREATE DATABASE statement`.
 
+   .. tab:: Sybase
+      :tabid: sybase-ddl
+
+      Importing DDL files with SybaseASE is not supported.
```

Comment:
```suggestion
      Importing DDL files with Sybase ASE is not supported.
```

Reply:
Sure @sarah-olson-mongodb, I have updated to `Sybase ASE` throughout for consistency.